v4/export/config.go

```diff
@@ -233,6 +237,7 @@ func (conf *Config) DefineFlags(flags *pflag.FlagSet) {
 	flags.Duration(flagReadTimeout, 15*time.Minute, "I/O read timeout for db connection.")
 	flags.MarkHidden(flagReadTimeout)
 	flags.Bool(flagTransactionalConsistency, true, "Only support transactional consistency")
+	flags.BoolP(flagCompress, "c", false, "Compress output files")
```

Comment:
is it possible to make this option take "gz" (or "gzip") as input? to allow for other compression formats

Reply:
addressed in https://github.com/pingcap/dumpling/pull/202/commits/8a5fe5de35a68dcb3dbe0b3620558a49bcf192a3